pkg/cmd/build/run.go

```diff
@@ -30,18 +30,20 @@ func RunBuildCmdLine(cmd *BuildCmd, fields *contracts.BuildInfo, msgs *[]string)
 		return msg.ErrorBuilding
 	}
 
+	var vulcanParams string
+
 	if fields.Preset != "" {
+		vulcanParams = " --preset " + fields.Preset
```

Comment:
What do you think about adding a check to avoid unnecessary empty space?  
```go
if vulcanParams != "" {
		vulcanParams += " "
}
```

Reply:
I'm not sure if I understand how is this helpful... 🤔